beacon-chain/powchain/service.go

```diff
@@ -185,6 +200,20 @@ func (w *Web3Service) Stop() error {
 func (w *Web3Service) ChainStartFeed() *event.Feed {
 	return w.chainStartFeed
 }
+func (w *Web3Service) saveToDb() {
+	powDepState := &pb.POWDepositState{}
+	if (w.depositTrie != nil && w.depositTrie != &trieutil.DepositTrie{}) {
+		(*powDepState).DepositTrie = w.depositTrie.ToProtoDepositTrie()
+	}
+	if w.LatestBlockHash().Bytes() != nil {
+		(*powDepState).LatestBlockHash = w.LatestBlockHash().Bytes()
+	}
+	if w.LatestBlockHeight() != nil {
+		(*powDepState).LastBlockHeight = w.LatestBlockHeight().Uint64()
+	}
+	(*powDepState).DepositCount = uint64(PrometheusToFloat64(validDepositsCount))
```

Comment:
I don't know if this is a good idea, we shouldn't be relying on the value of these metrics in our code

Reply:
What do you mean ?

Reply:
removed DepositCount